src/components/NewMessage/NewMessage.vue

```diff
@@ -644,14 +645,24 @@ export default {
 		},
 
 		async handleEdit() {
+
+			if (this.messageToEdit.messageParameters.length !== 0) {
+				// Captions editing is not supported yet
```

Comment:
But just remove it, it's coming before the final: https://github.com/nextcloud/spreed/pull/11371 (ready for review actually)

Reply:
As the PR is still draft, I just noted it there in to implement it after that API PR because I saw it ready for review.

Reply:
It's merged now :)